Shopist/Shopist/Scenes/DetailViewController.swift

```diff
@@ -0,0 +1,45 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import UIKit
+
+final class DetailViewController: ListViewController {
+    let category: Category
+    private var items = [Item]()
+
+    init(with category: Category) {
+        self.category = category
+        super.init(collectionViewLayout: UICollectionViewFlowLayout())
+    }
+
+    required init?(coder: NSCoder) { fatalError() }
+
+    override func fetch(with api: API) {
+        api.getItems(for: category) { result in
+            switch result {
+            case .success(let items):
+                self.items = items
+            case .failure(let error):
+                print(error)
+            }
+            self.reload()
+        }
+    }
+
+    override func config(cell: Cell, for index: Int) {
+        let item = items[index]
+        var text = "\(item.id): \(item.name)\n\(item.price)"
+        if item.isInStock {
+            text.append(" IN STOCK!")
+        }
```

Comment:
Presenting ID to the user looks weird. Let's keep the number of info the same as Android - so only product name:

![ref2](https://user-images.githubusercontent.com/2358722/89169994-544b3800-d57f-11ea-9726-6e7cf3e1aeae.png)